OpenRA.Game/Game.cs

```diff
@@ -51,13 +51,14 @@ public static class Game
 		public static Sound Sound;
 		public static bool HasInputFocus = false;
 
-		public static bool BenchmarkMode = false;
-
 		public static string EngineVersion { get; private set; }
 		public static LocalPlayerProfile LocalPlayerProfile;
 
+		public static bool ExitOnEnd = false;
+
 		static Task discoverNat;
 		static bool takeScreenshot = false;
+		static IBenchmark benchmark = new EmptyBenchmark();
```

Comment:
IMO the interface here isn't too useful - we can use `Benchmark` directly and default to `null` for normal games.